src/error.rs

```diff
@@ -626,6 +626,24 @@ impl From<UrlParseError> for UrlGenerationError {
     }
 }
 
+/// Errors which can occur when serving static files.
+#[derive(Fail, Debug, PartialEq)]
+pub enum StaticFileError {
+    /// Path is not a directory
+    #[fail(display = "Path is not a directory")]
```

Comment:
Please expand the error description: `Path is not a directory. Unable to serve static files`